test/Microsoft.AspNetCore.Razor.Test/CodeGenerators/CSharpTagHelperRenderingTest.cs

```diff
@@ -21,6 +21,62 @@ public class CSharpTagHelperRenderingTest : TagHelperTestBase
         private static IEnumerable<TagHelperDescriptor> PrefixedPAndInputTagHelperDescriptors { get; }
             = BuildPAndInputTagHelperDescriptors(prefix: "THS");
 
+        private static IEnumerable<TagHelperDescriptor> CSSSelectorTagHelperDescriptors
```

Comment:
Disagree since `name=""` is something tag helper authors may want to match. Should therefore support `[name='']` and there's no reason to subset the CSS spec to disallow the other empty value cases.